library/testing/test.h

```diff
@@ -0,0 +1,139 @@
+#ifndef Test_h
+#define Test_h
+
+#include <functional>
+#include <iostream>
+#include <sstream>
+
+class Test
```

Comment:
Let's find a class and filename that match the style.  
Maybe we can merge it with `TestSDKHelpers` too?

Reply:
agreed

Reply:
some doc would go a long way to understand what this do btw

Reply:
renamed, feel free to re-open if you have better naming ideas